### PR TITLE
implemented task and tests

### PR DESCRIPTION
**Pull request by ivadimko**

### CODE REVIEW

File: src/getOperands.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ *
+ * Function takes string which is Math expression of 2 numbers and returns it's
+ * operands in the next format: 'a: <first_operand>, b: <second_operand>'.
+ *
+ * Given string is a math operation which contains 2 numbers (operands) and
+ * operator (*, /, +, -). For example: '1 + 2',  '1.2 * -3.4',  '-3 / -6',
+ * '-2 - 2'
+ *
+ * Allowed operators: '+', '-', '*', '/'.
+ * Allowed whitespaces near operator and numbers.
```
Comment (roman-mate) [inline src/getOperands.js]: Whitespaces around operator and numbers are allowed.

File: src/getOperands.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ * Numbers can be floating (1.2, 2.354).
```
Comment (roman-mate) [inline src/getOperands.js]: floating -> floats.

File: src/getOperands.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+ * Negative number has '-' behind without whitespace (-2, -2.354).
```
Comment (roman-mate) [inline src/getOperands.js]: Negative numbers have a minus sign '-' before them without whitespace.

File: src/getOperands.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+/**
+ * Implement getOperands function:
+ *
+ * Function takes string which is Math expression of 2 numbers and returns it's
```
Comment (mgrinko) [inline src/getOperands.js]: `takes a string`

Comment (ivadimko) [reply src/getOperands.js]: pushed directly in `master`

**Pull request merged**